pyccl/cosmology.py

```diff
@@ -556,6 +572,12 @@ def _compute_nonlin_power(self):
         elif mps == 'emulator':
             pk = self.nl_pk_emu.get_pk2d(self)
 
+        # Include baryonic effects
+        if self.baryons is not None:
+            if self.baryons == 'bcm':  # Initialise convenience baryons model
+                self.baryons = baryons.BaryonsSchneider15(
+                    log10Mc=np.log10(1.2E14), eta_b=0.5, k_s=55.0)
```

Comment:
Are these the fiducial values? Do we need to pass them explicitly?

Reply:
yes, are there better values we should use as default? The idea is to provide 'bcm' as a quick option for someone who wants to check out what happens if you have baryons. Anyone who wants to properly deal with baryonic effects should create their own `Baryons` object and their associated parameters.

Reply:
@tilmantroester ok, so you're proposing we even remove the "bcm" option? That'd be fine with me.

Reply:
I'd be fine with that as well

Reply:
Done